ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/visitor/core/entities/method/AttributeInfo.java

```diff
@@ -0,0 +1,27 @@
+package edu.usb.argos.ASTProcessor.visitor.core.entities.method;
+
+import java.util.List;
+
+public class AttributeInfo {
```

Comment:
It would be good if you don't abbreviate the class name.

Reply:
Done